packages/js/uri-resolvers/jest.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Since we don't really have tests on this package, would it make sense to remove this file?

Reply:
Let's keep it, since we'll be adding tests in the near future :)